src/app/utils/constants.ts

```diff
@@ -8,39 +13,58 @@ export const LND_DIR = {
   LINUX: '~/.lnd/data/chain/bitcoin/*',
 };
 
+export enum CHAIN_TYPE {
+  BITCOIN = 'bitcoin',
+  LITECOIN = 'litecoin',
+}
+
+export const blockchainLogos: { [key in CHAIN_TYPE]: React.ComponentType<CustomIconComponentProps> } = {
```

Comment:
These actually aren't antd icons, they're React SVG components (As handled by `@svgr/webpack`.) A correct and simpler way to type this would be simply `React.ReactNode`.

Reply:
Thanks for the quick feedback! I’ll have this done tonight :)

Reply:
Typing blockchainLogos as React.ReactNode causes Travis CI to fail. I've set it back to React.ComponentType<CustomIconComponentProps> for now, as that was working well.